contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -551,16 +553,20 @@ List *handle_where_clause_attnums(ParseState *pstate, Node *w_clause, List *targ
 	if (nodeTag(w_clause) == T_A_Expr)
 	{
 		A_Expr *where_clause = (A_Expr *)w_clause;
+		ColumnRef *ref;
+		Value *field;
+		char *name;
+		int attrno;
```

Comment:
Seems like you need these variables in all of the individual blocks/conditions, better to declare it outside if possible and then free them at the end

Reply:
adressed